src/test/java/jenkins/plugins/git/GitSCMSourceTraitsTest.java

```diff
@@ -83,6 +83,60 @@ public void modern() throws Exception {
         assertThat(instance.getTraits(), is(Collections.<SCMSourceTrait>emptyList()));
     }
 
+    @Test
+    public void cleancheckout_v1() {
```

Comment:
I would name this `cleancheckout_v1_extension` to make it more clear.

Reply:
Done